examples/c/wind_turbine_nmpc.c

```diff
@@ -518,10 +518,10 @@ int main()
 	for (int i = 0; i < NN; i++)
 	{
 		plan->nlp_dynamics[i] = CONTINUOUS_MODEL;
-        // plan->sim_solver_plan[i].sim_solver = ERK;
-		// plan->sim_solver_plan[i].sim_solver = IRK;
-		// plan->sim_solver_plan[i].sim_solver = NEW_LIFTED_IRK;
-		plan->sim_solver_plan[i].sim_solver = GNSF;
+//		plan->sim_solver_plan[i].sim_solver = ERK;
+		plan->sim_solver_plan[i].sim_solver = IRK;
+//		plan->sim_solver_plan[i].sim_solver = NEW_LIFTED_IRK;
+//		plan->sim_solver_plan[i].sim_solver = GNSF;
```

Comment:
someone should clean up the examples since we test anyway all exotic combinations in the unit tests now..

Reply:
I would keep this stuff, this is how I found there was another bug in GNSF+BLASFEO

I don't trust unit tests alone...